src/attribution/onCLS.ts

```diff
@@ -25,11 +25,11 @@ import {
 } from '../types.js';
 
 const getLargestLayoutShiftEntry = (entries: LayoutShift[]) => {
-  return entries.reduce((a, b) => (a && a.value > b.value ? a : b));
+  return entries.reduce((a, b) => (a?.value > b.value ? a : b));
```

Comment:
how would this ever happen?

Reply:
Hmmm, I think you're right that it can't happen? My assumption is I originally thought that since we weren't passing an initial value as the second param, then it would be undefined, but looking at the reduce docs (and when testing) it seems like that's not true.